core/src/components/input/input.ionic.scss

```diff
@@ -140,7 +140,7 @@
 // ----------------------------------------------------------------
 
 .input-bottom {
-  @include padding(7px, 0);
+  @include padding(globals.$ionic-space-100, 0);
```

Comment:
After adding the ionic theme file to the `setContent`, the snapshots were trying to add more spacing than expected. After investigating, I noticed that this no longer matched Figma.